.github/workflows/build.yml

```diff
@@ -0,0 +1,12 @@
+name: build
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  build:
+    uses: jbanghub/.github/.github/workflows/shared-build.yml@main
```

Comment:
This workflow file just ensures that the JBang catalog scripts can resolve all its deps and compile.